### PR TITLE
add zsh style autocompletion docs

### PR DESCRIPTION
**Pull request by ecsecta**
## WHAT
Adding docs for Zsh autocompletion setup.

## WHY
Took me too long to setup and find the issues.

## Related issue (if exists)
#44 #32

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -94,6 +94,8 @@ For example, when you want to pass kubecolor result to grep (`kubecolor get pods
 kubectl provides [autocompletion feature](https://kubernetes.io/docs/tasks/tools/install-kubectl/#enable-kubectl-autocompletion). If you are
 already using it, you might have to configure it for kubecolor.
 
+#### Bash / Fish
```
Comment (dty1er) [inline README.md:97]: Did you make sure this works on fish? I don't have fish environment so I am not sure it.

File: README.md [hunk 2]
```diff
@@ -107,6 +109,17 @@ If you are using an alias like `k="kubecolor"`, then just change above like:
 complete -o default -F __start_kubectl k
 ```
 
+#### Zsh
+
+For zsh make sure this lines are present in your .zshrc file:
```
Comment (dty1er) [inline README.md:114]: Please fix this for consistency with another part.

```suggestion
For zsh make sure these lines are present in your zsh config file.
```

**Review by dty1er — CHANGES_REQUESTED** [review]
Thank you for sending this PR! Looks very promising.

File: README.md (hunk 1 above)
Comment (ecsecta) [reply README.md:97]: Ups, you are right, this was just an assumption, I will correct to 'Bash' only

File: README.md (hunk 2 above)
Comment (ecsecta) [reply README.md]: Updated these...

**Comment by dty1er** [comment]
Thank you! I also made sure the procedure works.
This is helpful a lot!!

**Review by dty1er — APPROVED** [review]

**Comment by ecsecta** [comment]
Thank you very much for this project!

**Pull request merged**